include/SDL3/SDL_audio.h

```diff
@@ -1113,6 +1113,40 @@ extern DECLSPEC void SDLCALL SDL_UnlockAudioDevice(SDL_AudioDeviceID dev);
  */
 extern DECLSPEC void SDLCALL SDL_CloseAudioDevice(SDL_AudioDeviceID dev);
 
+
+extern DECLSPEC void SDLCALL SDL_CloseAudioDevice(SDL_AudioDeviceID dev);
+ 
+/**
+ * Convert some audio data of one format to another format.
+ *
+ * \param src_format The format of the source audio
+ * \param src_channels The number of channels of the source audio
+ * \param src_rate The sampling rate of the source audio
+ * \param src_len  The len of src_data
+ * \param src_data The audio data to be converted
+ * \param dst_format The format of the desired audio output
+ * \param dst_channels The number of channels of the desired audio output
+ * \param dst_rate The sampling rate of the desired audio output
+ * \param dst_len  Will be filled with the len of dst_data
+ * \param dst_data Will be filled with a pointer to converted audio data, which should be freed with SDL_free().
+ *
+ * \returns 0 on success, or -1 on error.
```

Comment:
What happens to dst_data in the error case?

Reply:
this is not allocated and NULL
updated doc